### ISSUE TITLE
Value of the fine structure constant

### ISSUE THREAD

**Issue by lnmaurer** [hgrecco/pint#815]
I've run into what seems to be a problem with the definition of the fine structure constant while calculating the Bohr radius. I'm using version 0.9 installed through pip. It looks like there is a newer version of constants_en.txt in master, so maybe that fixes this issue.

Consider [the second definition](https://en.wikipedia.org/wiki/Bohr_radius#Definition_and_value) of the Bohr radius:

`In : ureg = pint.UnitRegistry(system='mks')`
`In : (1*ureg.hbar/ureg.m_e/ureg.c/ureg.fine_structure_constant).to_base_units()`                                                                     
`Out: 3.861592677119701e-13 <Unit('meter / fine_structure_constant')>`

I'm not sure that I would consider the fine structure constant as a base unit of mks, but in any case I can just convert it to meters instead:

`In  : (1*ureg.hbar/ureg.m_e/ureg.c/ureg.fine_structure_constant).to(ureg.meter)`                                                                                                               
`Out: 3.861592677119701e-13 <Unit('meter')>`

Both answers are the same. So, the fine structure constant in pint is 1? The correct value of the Bohr radius can be found by other means:

`In : (4*pi*ureg.epsilon_0*ureg.hbar**2/ureg.m_e/ureg.e**2).to_base_units()`                                                                                                                   
`Out: 5.2917726163680124e-11 <Unit('meter')>`

FWIW, the definition of the fine structure constant in master is unusual. It might be worth using a [more conventional definition](https://en.wikipedia.org/wiki/Fine-structure_constant#Definition).

**Comment by Jellby** [comment]
I guess that was fixed with #805. With the current version:

```
>>> (1*ureg.hbar/ureg.m_e/ureg.c/ureg.fine_structure_constant).to_base_units()
<Quantity(5.291772109039774e-11, 'meter')>

>>> (1*ureg.hbar/ureg.m_e/ureg.c/ureg.fine_structure_constant).to(ureg.meter)
<Quantity(5.291772109039774e-11, 'meter')>

>>> (4*ureg.pi*ureg.epsilon_0*ureg.hbar**2/ureg.m_e/ureg.e**2).to_base_units()
<Quantity(5.2917721090397735e-11, 'meter')>
```

The definition of the fine structure constant is intended to minimize the number of measured constants as well as the uncertainty of the value (when computed from the uncertainties of the other constants it's based on). In practice, as far as I could understand, one of the most accurately known constants is the Rydberg constant, so it makes sense to base the rest on it. However, the NIST page (https://physics.nist.gov/cuu/Constants/) gives circular definitions, that's the same problem as using "more conventional" definitions: we need the value of epsilon_0, or mu_0, or k_C, or Z_0... and those are all based on alpha (or on something else).

I was happy that, once I include the uncertainties of the measured constants, I could automatically obtain:
```
1 bohr = 5.29177210904(81)e-11 m
```
pretty close to the `5.29177210903(80)e-11 m` given by NIST.

**Comment by hgrecco** [comment]
Indeed this is fixed.
